tedana/stats.py

```diff
@@ -34,7 +93,7 @@ def getfbounds(n_echos):
     return f05, f025, f01
 
 
-def computefeats2(data, mmix, mask=None, normalize=True):
+def computefeats2(data, mmix, mask=None):
```

Comment:
I'd love to see this renamed. Perhaps `get_ls_zvalues`? It should also return standardized parameter estimates, but those are relatively unimportant compared to the z-values.